rero_ils/modules/cli.py

```diff
@@ -623,8 +623,8 @@ def do_worker(marc21records, results, pid_required):
         except Exception as err:
             msg = 'ERROR:\t{pid}\t{err}'.format(pid=pid, err=err.args[0])
             click.secho(msg, err=True, fg='red')
-            # import traceback
-            # traceback.print_exc()
+            if debug:
```

Comment:
is this related to this commit? Please add this, at least, in the commit message.

Reply:
Ok